### PR TITLE
IE table fix

### PR DESCRIPTION
**Pull request by talimarcus**
- implement fileTable using `<div>` instead of `<table>` so that it works with dom-repeat in IE11
- remove unnecessary dependencies and sass imports

### CODE REVIEW

File: sass/px-file-upload.scss [hunk 1]
```diff
@@ -27,42 +25,61 @@ $inuit-enable-btn--secondary: true;
...
 
 // Trumps
+$inuit-enable-paddings--tiny : true;
 $inuit-enable-paddings--small : true;
+$inuit-enable-paddings : true;
 $inuit-enable-paddings--large : true;
+$inuit-enable-paddings--huge : true;
 $inuit-enable-margins--tiny : true;
+$inuit-enable-margins--small : true;
+$inuit-enable-margins--large : true;
```
Comment (nonmetalhail) [inline sass/px-file-upload.scss:84]: Do you need all these padding and margins? I dont see many being used

File: px-file-upload.html [hunk 2]
```diff
@@ -34,31 +34,31 @@
...
-            <th>{{localize('Size')}}</th>
-            <th>{{localize('Type')}}</th>
-            <th></th>
-          </tr>
+        <div class="hidden file-upload-fileTable" id="fileTable">
+          <div class="file-upload-tr label">
+            <div class="file-upload-th">{{localize('Name')}}</div>
+            <div class="file-upload-th">{{localize('Size')}}</div>
+            <div class="file-upload-th">{{localize('Type')}}</div>
+            <div class="file-upload-th"></div>
```
Comment (nonmetalhail) [inline px-file-upload.html:45]: Why this last one that is empty?

**Review by nonmetalhail — CHANGES_REQUESTED** [review]
No test changes?

File: px-file-upload.html (hunk 2 above)
Comment (talimarcus) [reply px-file-upload.html:45]: that's to extend the header row to include the column with the X icon at the end of each row. looks like this without it:

screen shot 2017-08-11 at 2 28 47 pm

File: sass/px-file-upload.scss (hunk 1 above)
Comment (talimarcus) [reply sass/px-file-upload.scss:84]: all of these are being used -- some in the sass and some in the html (`u-p-`, `u-pb+`, etc)

**Comment by talimarcus** [comment]
@nonmetalhail tests all still passing

**Review by nonmetalhail — APPROVED** [review]

**Review by kngauriet — COMMENTED** [review]
It works perfect. Thank you very much.

**Pull request merged**